bloomstack_core/patches/v0_0_1/delete_bom_from_property_setter.py

```diff
@@ -0,0 +1,7 @@
+import frappe
+
+def execute():
+    # Delete BOM property from property setter.
+    frappe.db.sql("""DELETE FROM `tabProperty Setter`
+    WHERE
+    doc_type='BOM'""")
```

Comment:
You do not want to delete all BOM fields, just delete the field you are removing